src/applications/check-in/components/ErrorMessage.jsx

```diff
@@ -19,7 +19,32 @@ const ErrorMessage = ({ message, additionalDetails }) => {
       </va-alert>
     );
 
-  return (
+  return validationError ? (
```

Comment:
We should possibly wrap this update in the telephone feature flag, so that it all goes out together. May be worth asking Lori if this can go out on its own though. Other than that, I think things are looking good!

Reply:
Oh that's a great catch, this should definitely be wrapped.